### PR TITLE
Added Collection Location column on shipping page (741)

### PR DESCRIPTION
**Pull request by amber-emmes**
As per [741](https://github.com/episphere/connect/issues/741).

### CODE REVIEW

**Review by jhflorey — APPROVED** [review]
look good to me

File: src/pages/shipping.js [hunk 1]
```diff
@@ -163,11 +164,19 @@ const populateAvailableCollectionsList = async (availableCollectionsObj, loadFro
 
     for (const bagId of bagIdList) {
         if (bagId !== "unlabelled") {
+            const specimen = availableCollectionsObj[bagId].specimen;
             const rowEle = tableEle.insertRow();
             rowEle.insertCell(0).innerHTML = bagId;
             rowEle.insertCell(1).innerHTML = availableCollectionsObj[bagId].length;
+            if(specimen && specimen[conceptIds.collectionType] === conceptIds.clinical) {
```
Comment (we-ai) [inline src/pages/shipping.js:171]:
```suggestion
            if (specimen && specimen[conceptIds.collectionType] === conceptIds.clinical) {
```

File: src/pages/shipping.js [hunk 2]
```diff
@@ -163,11 +164,19 @@ const populateAvailableCollectionsList = async (availableCollectionsObj, loadFro
...
+                rowEle.insertCell(2).innerHTML = 'Clinical';
```
Comment (we-ai) [inline src/pages/shipping.js:172]: `textContent` has better performance.

**Review by we-ai — APPROVED** [review]
Commented on minor things.

**Pull request merged**